libraries/shared-gui/src/shared/gui/GLMHelpers.cpp

```diff
@@ -0,0 +1,21 @@
+//
+//  GLMHelpers.cpp
+//  libraries/shared-gui/src/shared/gui
+//
+//  Created by Stephen Birarda on 2014-08-07.
+//  Copyright 2014 High Fidelity, Inc.
```

Comment:
Is this correct?

Reply:
I just copied part of the code, so I think it makes sense to leave the copyright statement untouched. Technically I created the file, but it was just a part of moving things around, and not creating something new.